packages/floating-vue/nuxt.d.mts

```diff
@@ -0,0 +1,12 @@
+import type { defineNuxtModule } from '@nuxt/kit'
+import type { FloatingVueDirectivesOptions } from 'floating-vue/directives'
+import type { FloatingVueComponentsOptions } from 'floating-vue/components'
+
+export interface FloatingVueNuxtOptions {
+  directives?: FloatingVueDirectivesOptions
+  components?: FloatingVueComponentsOptions
+}
+
+const _default: ReturnType<typeof defineNuxtModule>
```

Comment:
review this, I guess we should add a `nuxt.ts` and generate the module properly...